### PR TITLE
Earthling updates

### PR DESCRIPTION
**Pull request by jpchip**
Forked Hive guidelines, updated minor details to be EI specific. Left as separate, public repo so it can be an easy reference for anyone in the company, whether or not they are using the FSSK. 

Resolves https://github.com/EarthlingInteractive/full-stack-starter-kit/issues/135

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -520,9 +509,9 @@ _Why:_
     _Why:_
     > This is a very well-known design to developers (your main API consumers). Apart from readability and ease of use, it allows us to write generic libraries and connectors without even knowing what the API is about.
 
-* use kebab-case for URLs.
+* use camelCase for URLs.
```
Comment (sbrudz) [inline README.md]: I prefer kebab-case for URLs because otherwise you have to worry about making your route checks case-insensitive.

**Comment by sbrudz** [comment]
They talk a lot about using a `develop` branch, which is akin to what we call `test`.  Should we adopt `develop` or change the wording to reflect current practices?

**Comment by jpchip** [comment]
I think we should adopt `develop`, `test` always make me think it's a deployment branch, like `stage`. 

I changed the url back to kebab, that makes sense to me.

**Comment by jpchip** [comment]
I just realized I would like to link to that node best practices doc and some of those code review articles, but I can do that in another PR. Let me know if this looks ok otherwise.

**Comment by sbrudz** [comment]
Yes, this looks good

**Pull request merged**